hadoop-yarn-project/hadoop-yarn/hadoop-yarn-csi/pom.xml

```diff
@@ -26,7 +26,8 @@
     <packaging>jar</packaging>
 
     <properties>
-        <grpc.version>1.26.0</grpc.version>
+        <grpc.version>1.53.0</grpc.version>
+        <animal-sniffer.version>1.21</animal-sniffer.version>
```

Comment:
this would require a change in LICENSE-binary file as well, right? it has 1.17 mentioned
```
org.codehaus.mojo:animal-sniffer-annotations:1.17
```